mne/utils/docs.py

```diff
@@ -2499,13 +2499,31 @@
     If it is a 2-tuple of minimum and maximum limit, then those
     physical ranges will be used. Only used for exporting EDF files.
 """
+docdict['export_params_add_ch_type'] = """
+add_ch_type : bool
+    Whether to add the channel type of the channel currently set
+    (default is False) to the signal label. Only used for
+    exporting EDF files.
+"""
 docdict['export_eeglab_note'] = """
 For EEGLAB exports, channel locations are expanded to full EEGLAB format.
 For more details see :func:`eeglabio.utils.cart_to_eeglab`.
 """
 docdict['export_edf_note'] = """
-For EDF exports, only EEG, ECoG and sEEG data are supported. In
-addition, EDF does not support storing a montage.
+For EDF exports, only channels that are measured in terms of Voltage are
+supported. For example, 'eeg', 'ecog', 'seeg', 'bio', 'emg', 'eog',
+'ecg', 'dbs', 'bio' data are supported. However, EDF files do not
+require the storage of channel types, so when re-reading the file
+naively, you will only get 'eeg' types by default. For example, if you
+have 'misc' channels, they will be read back in as 'eeg' type. If you
+want to set the channel types, then set the ``add_ch_type`` keyword.
+This will then for example take EEG channel ``Fz1`` and export the
+signal label as ``EEG Fz1`` in the EDF file. In order to properly set
```

Comment:
```suggestion
This will then for example take EEG channel ``Fz`` and export the
signal label as ``EEG Fz`` in the EDF file. In order to properly set
```
Let's use a realistic channel name in our example :)